hyperledger/iroha
# v1.3: support/1.2.x with fixed DCO to main

- Batch db requests (#791)
- fixup! no clone (#796)
- Yac: do not consider next round as Future (#742)
- add wsv restore mode - sync blockstore (#812)
- Feature/Improves stability in networks with faulty nodes (#809)
- fix/Block request timeout. (#840)
- [DOPS-1289] Rename master branch (#804)
- Feature/subscription engine (#813)
- Subscription engine singleton (#849)
- Add prometheus metrics sample (#841)
- clang fix (#909)
- 881 no --drop-state flag for first run (#900)
- Infrastructure: increase Windows CI disk size
- vcpkg: add pkg-config patch
- Infrastructure: use main branch where required
- build fix
- OnDemandOrderingServiceImpl: remove boost::adaptors::filter usage
- SE features update
- Fix postgres_options_test
- Clang-format
- [SE] do not allow stack-created subscriber
- AmetsuchiTest: add condition for prepared block tests
- [SE] sync/async dispatcher
- added schedulers bind/unbind additional schedulers to dispatcher
- [RDB] rocksdb init
- Initialize default log manager to report config-file issues.
- fixed links
- configuration information+small fixes+change in m2r extension
- PR template
- reinterpret_pointer_cast moved to iroha ns
- [GRPC] Deny messages longer than 4MB
- irohad flags + link fixes
- Irohad: add subscription engine field
- TransactionProcessor: decouple from rxcpp
- BlockLoader: replace observable with any_range
- Synchronizer: decouple from rxcpp
- Fix fuzzing build
- Simulator: decouple from rxcpp
- BlockLoader: replace range with BlockReader
- Review fixes, rename methods
- Add missing deduction guide, apply clang-format-7
- Implement make_weak, replace monostate with dedicated struct
- Add missing include
- FlatFile: close file before rename
- [RDB] wsv commands and queries
- Docs: Contributing Guide Fix (#873)
- Create repolinter.yml
- fixing python example (#895)
- Missing docs from master (#892)
- fixed doc build (#902)
- Infrastructure: increase Windows CI disk size
- vcpkg: add pkg-config patch
- Replace round delay with proposal creation timeout
- Ordering: send batches to current and next rounds
- YacGateImpl: decouple from rxcpp
- Yac: decouple from rxcpp
- Ordering: decouple from rxcpp, async proposal request
- Fix build, add proposal check
- Review fixes
- Build fixes
- Yac: format namespace usage
- Fix consensus sunny day test
- [RDB] block indexer
- [RDB] command executor
- [Docs] Option pg_config is deprecated #1037
- Metrics all in one (#943)
- fix precision
- [RDB] read precision in SubstractAssetQuantity
- Synchronizer: fix assert failure
- [RDB] specific query executor
- Remove rxcpp from ametsuchi, torii, pending txs storage; make gRPC params optional
- [CI][GHActions] Introduce GitHub Actions Iroha1 (#1028)
- Docs: Block Path Config Description (#1080)
- [GHActions][Hotfix] Better matrix generating - fix red cross (#1039)
- [GHA] gha-branches and -DCMAKE_BUILD_TYPE from matrix (#1157)
- [SE] fixed thread poll tasks balancer
- Remove storage from peer orderer
- Add commands to burrow integration (#1152)
- [RDB] integration
- [client-api] extending TxPaginationMeta in GetAccountTransactions, GetAccountAssetsTransactions (#1092)
- Add scripts/fix-dco.sh
